### ISSUE TITLE
License question

### ISSUE THREAD

**Issue by DTraitor** [Theawesomeboophis/UndeadPeopleTileset#11]
I'd like to use those sprites in other game (we are making something simillar to CDDA in SS13). Under what license are this tileset?

**Comment by Theawesomeboophis** [comment]
The tileset is under the CC-BY-SA 3.0 license, same as the game as far as I know. Although if sprites are used, the link to the tileset repo and this must be put somewhere, as the license requires attribution to be used. I must also warn you that some sprites in this tileset come from unverified sources and may be infringing on copyright, I am working to rectify this and track down the sources of sprites. I will reach out to Erk, the creator of Ultica as I believe his tileset may better aesthetically blend with SS13 and has no unverified sources of sprites.

**Comment by I-am-Erk** [comment]
Yes, if you're concerned about avoiding any licensure issues, there are numerous known pirated tiles in this set that could cause problems. Any set in the main cataclysm repository is under cc-by-sa 4.0 and wouldn't have that problem (and can be used without any special dispensation as long as appropriate credit is used and licensure requirements kept)
